package.json

```
@@ -9,7 +9,8 @@
     "start": "npm run build && node dist/index.js",
     "dev": "nodemon",
     "format": "prettier --write 'src/**/*.ts'",
-    "lint": "eslint 'src/**/*.{js,ts}'"
+    "lint": "eslint 'src/**/*.{js,ts}'",
+    "migrate:deploy": "npx prisma migrate deploy"
```

```suggestion
    "migrate:deploy": "npm prisma migrate deploy"
```
para garantir que ta usando o que ta instalado no node_modules e nao instalar global sem necessidade